src/ragapp/backend/routers/chat/index.py

```diff
@@ -84,3 +68,17 @@ async def chat_request(
         result=Message(role=MessageRole.ASSISTANT, content=response.response),
         nodes=SourceNodes.from_source_nodes(response.source_nodes),
     )
+
+
+def process_response_nodes(
+    nodes: List[NodeWithScore],
+    background_tasks: BackgroundTasks,
+):
+    try:
+        # Start background tasks to download documents from LlamaCloud if needed
+        from app.engine.service import LLamaCloudFileService
```

Comment:
As we always call `from app.engine.service import` so the LLamaCloudFileService file must be there, so patching is the only way to go.